docs/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Adjust cve.org collector to correctly label CISA-ADP CVSS scores as CISA (OSIDB-4204)
+- Adjust JIRA tracker generator to include NIST/CISA CVSS scores when RH ones
+  don't meet mod7 criteria (OSIDB-4199)
```

Comment:
nitpick: I am not sure `mod7` is a generally understood term. I personally see it for the first time. Though with an SLA policy knowledge it should be clear what it means